### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by DaniGenerous** [community/community#47805] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello,

I was hoping someone might be able to help me resolve an issue with my github page. I want to display a background picture called "triangles.jpeg" but it does not show up on my published page. On my browser everything appears fine. I have played with case sensitivity as well as specifying the images location in a folder but I cannot get it to appear on my page.

I've viewed the similar posts, however, I feel as though I've tried all the recommendations given and considering how new I am with html/css I'm struggling to pinpoint the error myself. 

Any help would be much appreciated.

**Comment by radar-blip** [comment, marked as answer]
I dont see the triangles.jpeg in your code anywhere. 
Not in any of your index.html, it should be there somewhere in order for it to display.

**Reply by DaniGenerous** [reply]
Since it's a decorative background I have it in my CSS.

**Reply by radar-blip** [reply]
Okay, you have it in a folder called images, but no such folder exists.
So, just bring it out into root.

Should be like so: `background-image: url("triangles.jpeg"); `

**Reply by DaniGenerous** [reply]
Ah! What a silly mistake. Thank you so very much!

**Reply by radar-blip** [reply]
You're welcome! 👍🏻
